src/assets/kongponents-theme.scss

```diff
@@ -97,13 +98,17 @@
     background: var(--section_colors-body) !important;
   }
 }
-
-.k-button:disabled, .kbutton[disabled] {
+#app .k-button {
+  &:disabled,
+  &[disabled] {
   opacity: .3;
   cursor: not-allowed;
+    color: var(--button_colors-primary-text) !important;
+    background-color: var(--button_colors-primary-fill) !important;
```

Comment:
Do we want to override disabled buttons with the primary button colors? 🤔

Reply:
Ahh, yes. It is hard-coded right now to be gray on gray.
<img width="415" alt="image" src="https://github.com/Kong/konnect-portal/assets/2126677/6f7ce0b1-db46-4c15-afe5-826536010eaa">

New one is the same primary color and fill, and just changes opacity and cursor to prevent interactivity.